### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by vibhansa-msft**
- [#999](https://github.com/Azure/azure-storage-fuse/issues/999) Upgrade dependencies to resolve known CVEs.
- [#1002](https://github.com/Azure/azure-storage-fuse/issues/1002) In case version check fails to connect to public container, dump a log to check network and proxy settings.
- [#1006](https://github.com/Azure/azure-storage-fuse/issues/1006) Remove user and group config from logrotate file.

### CODE REVIEW

File: .github/workflows/trivy.yaml [hunk 1]
```diff
@@ -31,23 +31,23 @@ jobs:
...
-          cat trivy-results.sarif
+      # - name: List Issues
+      #   run: |
+      #     cat trivy-results.sarif
 
-      - name: Upload Trivy scan results to GitHub Security tab
-        uses: github/codeql-action/upload-sarif@v2
-        with:
-          sarif_file: 'trivy-results.sarif'
+      # - name: Upload Trivy scan results to GitHub Security tab
```
Comment (andyzhangx) [inline .github/workflows/trivy.yaml:47]: it's not necessary to update, remove this step

File: .github/workflows/trivy.yaml [hunk 2]
```diff
@@ -31,23 +31,23 @@ jobs:
...
 
-      - name: Run Trivy vulnerability scanner in repo mode
-        uses: aquasecurity/trivy-action@master
-        with:
-          scan-type: 'fs'
-          ignore-unfixed: true
-          format: 'sarif'
-          output: 'trivy-results.sarif'
-          severity: 'LOW,MEDIUM,HIGH,CRITICAL'
+      # - name: Run Trivy vulnerability scanner in repo mode
```
Comment (andyzhangx) [inline .github/workflows/trivy.yaml:34]: you need to build blobfuse2 before this step: `Run Trivy vulnerability scanner in repo mode`

Comment (vibhansa-msft) [reply .github/workflows/trivy.yaml:34]: Removed this and added binary scan towards the end of this file

File: blobfuse2-ci.yaml [hunk 3]
```diff
@@ -51,6 +51,11 @@ jobs:
         sudo apt-get install $(fuselib) -y
       displayName: 'Install libfuse'
 
+    - script: |
```
Comment (gapra-msft) [inline blobfuse2-ci.yaml:54]: Do we need to run this here if we have it running on Workflows?

**Review by gapra-msft — COMMENTED** [review]
left a comment

File: .github/workflows/trivy.yaml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
+    
+    name: Build
+    runs-on: "ubuntu-22.04"
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      # - name: Run Trivy vulnerability scanner in repo mode
```
Comment (gapra-msft) [inline .github/workflows/trivy.yaml:34]: This can be removed right? The commented stuff?

Comment (vibhansa-msft) [reply .github/workflows/trivy.yaml:34]: done

File: blobfuse2-ci.yaml (hunk 3 above)
Comment (vibhansa-msft) [reply blobfuse2-ci.yaml:54]: workflows was not working earlier so thought of adding it here. Also the way that workflow gives output and this step gives is little different, so thought of adding it at both the places. But yes once workflow stablize we can just remove this part.

**Comment by vibhansa-msft** [comment]
@andyzhangx : kindly review this and provide your feedback.

**Review by andyzhangx — APPROVED** [review]
`trivy.yaml` lgtm, thanks.

**Comment by andyzhangx** [comment]
btw, pls also squash commits into a few commits, make commits history tidy

**Review by souravgupta-msft — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,6 @@
+## 2.0.2 (WIP)
+- Upgrade dependencies to resolve trivy scan failures.
```
Comment (gapra-msft) [inline CHANGELOG.md:2]:
```suggestion
- Upgrade dependencies to resolve known CVEs.
```

Comment (gapra-msft) [inline CHANGELOG.md:2]: Should we also add - logs issue if proxy is not set via environment? For your fix below?

**Review by gapra-msft — APPROVED** [review]

**Review by tasherif-msft — APPROVED** [review]

**Pull request merged**